native/Avalonia.Native/src/OSX/window.mm

```diff
@@ -1089,7 +1097,15 @@ - (bool) ignoreUserInput
 - (void)mouseEvent:(NSEvent *)event withType:(AvnRawMouseEventType) type
 {
     if([self ignoreUserInput])
+    {
```

Comment:
I think it's better to have this code inside of `ignoreUserInput` itself.

Reply:
fixed.